.travis.yml

```diff
@@ -0,0 +1,40 @@
+notifications:
+  email: false
+
+language: node_js
+node_js:
+  - 9
+  - 8
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+    # See https://github.com/atom/ci
+    - libsecret-1-dev
```

Comment:
Sorry for the nitpickyness, but this should be indented.

Reply:
Done!